modules/clients/src/test/java/org/apache/ignite/jdbc/JdbcErrorsAbstractSelfTest.java

```diff
@@ -719,6 +719,12 @@ public void testDdlWrongColumnName() throws SQLException {
 
         checkSqlErrorMessage("alter table test drop column wrong", "42000",
             "Failed to parse query. Column \"WRONG\" not found");
+
+        checkSqlErrorMessage("create table test(id integer primary key, AgE integer, AGe integer)", "42000",
+            "Duplicate column name: AGE");
+
+        checkSqlErrorMessage("create table test(\"id\" integer primary key, \"age\" integer, \"age\" integer)", "42000",
+            "Duplicate column name: age");
```

Comment:
The same as for IgniteCacheSqlQueryErrorSelfTest.java

Reply:
Done.